src/config.nim

```diff
@@ -41,7 +41,10 @@ macro parseObject(obj: typed, key, val: string) =
 
 proc loadConfig*(filePath: string): Config =
   result = defaultConfig
-  for line in filePath.lines:
+  for rawLine in filePath.lines:
+    let line = rawLine.strip
```

Comment:
whats the point of stripping the line separately if both components are then stripped anyway?

Reply:
To support cases like:

```
     # foo = bar
```